config.go

```diff
@@ -538,6 +543,14 @@ type Config struct {
 	//
 	// Required: defaults to false
 	SkipTargetVerification bool
+
+	// ReadTimeout is used to configure the MySQL client timeout for waiting for data from server.
+	// Timeout is in seconds. Defaults to unlimited.
+	ReadTimeout uint64
```

Comment:
If these are already in DatabaseConfig, do we still need to have it on the top level here?

Reply:
Oh didn't think about DatabaseConfig being exposed. That is indeed neat but JSON load lacks Duration support and we need to move the casting.